### PR TITLE
Changelog v2

### PR DESCRIPTION
**Pull request by michael-etzkorn**
**Related issue**: 
https://github.com/chipsalliance/rocket-chip/issues/2960

**Type of change**:  other enhancement

**Impact**: no functional change

**Development Phase**: implementation

**Release Notes**

This is part of the release notes! This changelog is aimed to help with migration and informing users between releases of RC.

### CODE REVIEW

**Comment by michael-etzkorn** [comment]
I'll take one more look ove. Then, we should be able to go ahead and merge. Minor touch-ups:
- [ ] add some details for the future release
- [ ] add a release date for v1.6.0

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+* As Scala FIRRTL Compiler (SFC) is being sunset, update chisel2 legacy code to chisel3 (https://github.com/chipsalliance/rocket-chip/pull/3097)
+
+* Removal of `RocketTilesKey` and `RocketCrossingKey` (https://github.com/chipsalliance/rocket-chip/pull/3133)
+
+## Version 1.6.0
+
+10 Oct 2022
+
+### Added
+* Support for Chisel 3.5.3 (https://github.com/chipsalliance/rocket-chip/pull/2931, https://github.com/chipsalliance/rocket-chip/pull/2937, https://github.com/chipsalliance/rocket-chip/pull/2947, https://github.com/chipsalliance/rocket-chip/pull/3005)
```
Comment (jackkoenig) [inline CHANGELOG.md:19]:
```suggestion
* Bump to Chisel 3.5.3 (https://github.com/chipsalliance/rocket-chip/pull/2931, https://github.com/chipsalliance/rocket-chip/pull/2937, https://github.com/chipsalliance/rocket-chip/pull/2947, https://github.com/chipsalliance/rocket-chip/pull/3005)
```

I don't have any real opinion on what sort of categorization y'all adopt, but I think it's important to call out that any bump of Chisel major versions is a major one for users of rocket-chip. It is a very *good* thing, but it does mean that they will also have to bump Chisel, so it's a big change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,451 @@
+
+# Changelog
+
+## Future Release
+
+Some highlights to be included in the next release:
+
+* Bump to chisel 3.5.4 (https://github.com/chipsalliance/rocket-chip/pull/3105)
+
+* As Scala FIRRTL Compiler (SFC) is being sunset, update chisel2 legacy code to chisel3 (https://github.com/chipsalliance/rocket-chip/pull/3097)
```
Comment (jackkoenig) [inline CHANGELOG.md:10]: Both of these are important coming changes, but technically the removal of Chisel 2 compatibility mode is unrelated to the sunsetting of SFC, they're just both slated to be marked deprecated in Chisel 3.6.

**Review by jerryz123 — APPROVED** [review]

**Comment by sequencer** [comment]
Thank you!!!

**Pull request merged**